app-android-xamarin/Moodis/Moodis/Feature/Group/GroupListAdapter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Support.V7.Widget;
+using Android.Views;
+using Android.Widget;
+
+namespace Moodis.Feature.Group
+{  
+    class GroupListAdapter : RecyclerView.Adapter
+    {
+        List<Group> groups;
+        Context context;
+        GroupActivityModel groupActivityModel = new GroupActivityModel();
+
+        public GroupListAdapter(Context context,List<Group> groups)
+        {
+            this.groups = groups;
+            this.context = context;
+        }
+
+        public override int ItemCount
+        {
+            get { return groups.Count(); }
+        }
+
+        public override void OnBindViewHolder(RecyclerView.ViewHolder holder, int position)
+        {
+            var viewHolder = holder as GroupViewHolder;
+            viewHolder.GroupNameLabel.Text = groups[position].Groupname;
+            viewHolder.GroupNameLabel.Click += (sender, e) =>
+            {
+                var usersInGroup = groups[position].Members;
+                var MyIntent = new Intent(context, typeof(GroupMembersActivity));
+                MyIntent.PutExtra("clicked", position);
```

Comment:
make "clicked" as static const string in group list activity named as `EXTRA_CLICKED_GROUP`